src/array_api_stubs/_draft/array_object.py

```diff
@@ -645,9 +641,6 @@ def __gt__(self: array, other: Union[int, float, array], /) -> array:
         """
         Computes the truth value of ``self_i > other_i`` for each element of an array instance with the respective element of the array ``other``.
 
-        .. note::
-           For backward compatibility, conforming implementations may support complex numbers; however, inequality comparison of complex numbers is unspecified and thus implementation-dependent (see :ref:`complex-number-ordering`).
-
```

Comment:
I'd suggest to keep the link to https://data-apis.org/array-api/latest/design_topics/complex_numbers.html#complex-number-ordering, here and in other comparison operators.

Rationale:on a first read, I was not sure if ordering for complex value was even defined in the standard.